sdk/attestation/attestation/src/attestationClient.ts

```diff
@@ -112,6 +156,101 @@ export class AttestationClient {
     return this._client;
   }
 
+  /** Attests an OpenEnclave report generated from an SGX Enclave using the OpenEnclave SDK.
+   * 
+   * @param report An OpenEnclave report generated by an SGX enclave.
+   * @param options Operation options for the attestOpenEnclave API call.
+   * @returns 
+   */
+  public async attestOpenEnclave(report : Uint8Array, options : AttestOpenEnclaveOptions={}) {
```

Comment:
can we make the return type for this function explicit? it looks funny in the API review.

Reply:
Oops, totally missed that one :).